.bazelci/presubmit.yml

```diff
@@ -3,15 +3,21 @@ platforms:
   ubuntu1404:
     build_targets:
     - "..."
+    - "@io_bazel_rules_go//..."
```

Comment:
Let's test `@io_bazel_rules_go//...` in here and `//...` in Travis CI.

I'd like to have coverage for both, but I don't want to make the presubmit 2x slower.

Ideally Bazel would recognize these as the same targets :(

Reply:
It looks like this is still going to test everything twice in BuildKite.